src/StripeTests/Infrastructure/test_data/TestService.cs

```diff
@@ -0,0 +1,16 @@
+namespace StripeTests.Infrastructure.test_data
```

Comment:
Can you rename this to `TestData` instead (and also change the directory's name)? I don't think the use of snake case in namespaces is idiomatic in C#.

Reply:
fixed!